### DISCUSSION TITLE
Custom size of Todo App

### DISCUSSION THREAD

**Discussion by danielcontro** [rxhanson/Rectangle#1415] [Ideas]
I was wondering if there was a way for specifying a custom "Rectangle size" for the Todo App, for example I find myself most of the time wanting to have a maximized window and would like to have my Todo App hovering over it in a Top Right fashion.
Actually I also wonder if it's possible at all to have a not focused window over a focused one. In case I'd love to work on the feature.
In my opinion it would also be nice to be able to have multiple todo apps as #348 but being able to cycle between them instead of having multiple windows at the same time, like something on the lines of iPad's Slide Over.

**Comment by rxhanson** [comment]
Thanks for looking to contribute. There is no capability exposed within macOS to "float" windows belonging to a different app above other windows, but I'm sure there are plenty of ways to improve this functionality regardless. I see no reason to turn down a good pull request here.
